src/components/EditPlayer/EditPlayer.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from "react";
+import { Link } from "react-router-dom";
+
+import "./editPlayer.css";
+import Button from "../Button";
+import PlayerForm from "../PlayerForm/PlayerForm";
+import { GameContext } from "../../contexts/gameContext";
+
+const EditPlayer = props => {
+  const { id } = props.match.params;
+  const game = useContext(GameContext);
+  const [players, setPlayers] = game.usePlayers;
+  const [playerToEditId] = useState(id);
+
+  const handlePlayerUpdate = player => {
+    const newData = [...players];
+
+    if (player.id) {
```

Comment:
I think this `if` can be removed, this was back when you had the create/edit logic together, right?